shtdlib.sh

```
@@ -157,8 +157,33 @@ function readlink_m {
     if [ "${#args[@]}" -gt 1 ] ; then
```

We should return an error if no params are provided.